frontend/public/components/deployment.jsx

```diff
@@ -32,8 +33,29 @@ const UpdateStrategy = (kind, deployment) => ({
   callback: () => configureUpdateStrategyModal({deployment}),
 });
 
+const pause = deployment => {
```

Comment:
Let's not copy and paste this. It would be better to define it in one place and use it both for deployment configs and deployments.